corehq/apps/hqadmin/templates/hqadmin/app_build_timings.html

```diff
@@ -0,0 +1,47 @@
+{% extends "hqwebapp/base.html" %}
```

Comment:
This should extend `hqwebapp/base_section.html` so it gets the basic menu and breadcrumbs. Almost nothing should extend `base.html` directly.

Reply:
Just switched it, though it doesn't change the appearance of the page at all.  It's currently not a domain page and doesn't have the site map info in the context.